@navikt/core/react/src/layout/bleed/Bleed.stories.tsx

```diff
@@ -0,0 +1,364 @@
+                <BodyLong>marginBlock=&quot;10 0&quot;</BodyLong>
+              </Box>
+            </Bleed>
+          </Box>
+        </Box>
+        <Box background="surface-alt-1-subtle" padding="5">
+          <Box background="surface-alt-2-subtle" padding="5">
+            <Bleed marginBlock="0 10">
+              <Box className="p" background="surface-success-subtle">
+                <HStack className="centered" justify="center" align="center">
+                  <Box className="testbox">marginBlock=&quot;0 10&quot;</Box>
+                </HStack>
+              </Box>
+            </Bleed>
+          </Box>
+        </Box>
+      </VStack>
+    </>
+  ),
+};
+
+export const AsChild = {
+  render: () => (
+    <>
+      {style}
+      <VStack gap="2">
+        <Box background="surface-alt-1-subtle" padding="5">
+          <Box background="surface-alt-2-subtle" padding="5">
+            <Bleed marginInline="10 0" asChild>
+              <Box className="p" background="surface-success-subtle">
+                <BodyLong>marginInline=&quot;10 0&quot;</BodyLong>
+              </Box>
+            </Bleed>
+          </Box>
+        </Box>
+        <Box background="surface-alt-1-subtle" padding="5">
+          <Box background="surface-alt-2-subtle" padding="5">
+            <Bleed marginInline="0 10" asChild>
+              <Box className="p" background="surface-success-subtle">
+                <BodyLong>marginInline=&quot;0 10&quot;</BodyLong>
+              </Box>
+            </Bleed>
+          </Box>
+        </Box>
+        <Box background="surface-alt-1-subtle" padding="5">
+          <Box background="surface-alt-2-subtle" padding="5">
+            <Bleed marginBlock="10 0" asChild>
+              <Box className="p" background="surface-success-subtle">
+                <BodyLong>marginBlock=&quot;10 0&quot;</BodyLong>
+              </Box>
+            </Bleed>
+          </Box>
+        </Box>
+        <Box background="surface-alt-1-subtle" padding="5">
+          <Box background="surface-alt-2-subtle" padding="5">
+            <Bleed marginBlock="0 10" asChild>
+              <Box className="p" background="surface-success-subtle">
+                <HStack className="centered" justify="center" align="center">
+                  <Box className="testbox">marginBlock=&quot;0 10&quot;</Box>
+                </HStack>
+              </Box>
+            </Bleed>
+          </Box>
+        </Box>
+      </VStack>
+    </>
+  ),
+};
+
+export const Breakpoints = {
+  render: () => (
+    <>
+      {style}
+      <VStack gap="2">
+        <Box background="surface-alt-1-subtle" padding={{ xs: "5", md: "10" }}>
+          <Box
+            background="surface-alt-2-subtle"
+            padding={{ xs: "5", md: "10" }}
+          >
+            <Bleed marginInline={{ xs: "10 0", md: "20 0" }}>
+              <Box className="p" background="surface-success-subtle">
+                <BodyLong>marginInline=&quot;10 0&quot;</BodyLong>
```

Comment:
Burde teksten oppdateres til å reflektere hele objektet sendt inn i `marginInline`? Samme under.

Feks.:
```suggestion
                <BodyLong>marginInline={ xs: &quot;10 0&quot;, md: &quot;20 0&quot; }</BodyLong>
```